### ISSUE TITLE
[Bug] Broken type imports in `dist`

### ISSUE THREAD

**Issue by KiwiKilian** [visgl/react-maplibre#5]
### Description

These [barrel file imports](https://github.com/visgl/react-maplibre/blob/master/src/components/marker.ts#L7-L13) break on build. E.g. when the package is installed, the file `node_modules/@vis.gl/react-maplibre/dist/components/marker.d.ts` has the following type imports:

```ts
import type { MarkerEvent, MarkerDragEvent, PopupInstance, MarkerInstance, MarkerOptions } from "../types.js";
```

Explore the distributed code [here](https://www.npmjs.com/package/@vis.gl/react-maplibre?activeTab=code).

So

```
import type { ... } from '../types'
```

gets currently transpiled to

```
import type { ... } from '../types.js'
```

Which probably happens due to these [tsconfig plugins](https://github.com/visgl/react-maplibre/blob/master/tsconfig.json#L13-L22)? 

### Expected Behavior

Types probably shouldn't be imported from `js` files.

### Steps to Reproduce

`event` has type `any` in the following:

```tsx
<Marker
  onClick={(event) => { /* ... */ }
/>                    
```

### Environment

- Framework version: 1.0.0-alpha.4
- Map library: MapLibre 4.7.1
- Browser:
- OS:

**Comment by Pessimistress** [comment]
This should be fixed in `1.0.0-alpha.5`.

**Comment by KiwiKilian** [comment]
Works fine now, thank you!

Just a quick note, [while alpha 4 was released as "latest", alpha 5 is no released as "beta"](https://www.npmjs.com/package/@vis.gl/react-maplibre?activeTab=versions) (which also doesn't match).
